external/fv3fit/tests/reservoir/test_model_adapter.py

```diff
@@ -34,10 +34,15 @@ def decode(self, latent_x):
 
 
 def get_initialized_hybrid_model():
+    # expects rank size (including halos) in latent space
+    divider = RankDivider((2, 2), ["x", "y"], [8, 8], 2)
+    autoencoder = DoNothingAutoencoder(6)
+    input_size = 6 * 6 * autoencoder.n_latent_dims
+    hybrid_input_size_per_subdomain = (
+        divider.subdomain_xy_size_without_overlap ** 2 * autoencoder.n_latent_dims
+    )
+    output_size = hybrid_input_size_per_subdomain
```

Comment:
These are the same values as before, just some edits from while I was trying to debug what broke in the adapter.